src/main/java/com/jfrog/ide/idea/configuration/GlobalSettings.java

```diff
@@ -203,18 +207,32 @@ public boolean areArtifactoryCredentialsSet() {
     private void migrateCredentialsFromFileToPasswordSafe(ServerConfigImpl serverConfig) {
         this.serverConfig.setUsername(serverConfig.getUsername());
         this.serverConfig.setPassword(serverConfig.getPassword());
+        this.serverConfig.setAccessToken(serverConfig.getAccessToken());
         this.serverConfig.addCredentialsToPasswordSafe();
         Application application = ApplicationManager.getApplication();
         application.invokeLater(application::saveSettings);
     }
 
     /**
-     * Determine whether should perform credentials migration or not.
+     * Determine whether we should perform credentials migration or not.
      *
      * @param xrayConfig - configurations read from 'jfrogConfig.xml'.
      * @return true if credentials are stored in file.
      */
     private boolean shouldPerformCredentialsMigration(ServerConfigImpl xrayConfig) {
-        return !isAnyBlank(xrayConfig.getUsername(), xrayConfig.getPassword());
+        return isNoneBlank(xrayConfig.getUsername(), xrayConfig.getPassword());
+    }
+
+    public void loadConnectionDetailsFromJfrogCli() {
+        // Try to read connection details using JFrog CLI only if no server is already configured.
+        if (areXrayCredentialsSet()) {
+            return;
+        }
+        try {
+            String configured = serverConfig.readConnectionDetailsFromJfrogCli() ? "Successfully" : "Couldn't";
```

Comment:
Doesn't ```serverConfig.readConnectionDetailsFromJfrogCli``` return a boolean?

Reply:
This way it works. Do you prefer a standard if and duplicated log message print?